consensus/consensus_validator.go

```diff
@@ -66,6 +67,7 @@ func (consensus *Consensus) processAnnounceMessage(message *msg_pb.Message) {
 
 	copy(consensus.blockHash[:], blockHash[:])
 	consensus.block = block
+	consensus.BlockVerifier = func(*types.Block) error { return nil }
```

Comment:
Is this WIP code?

Reply:
Yes, please take a look at the overall direction; the finishing touches are not in.

Reply:
Fix this; it shouldn't be here

Reply:
this is the equivalent of onAnnounce